dataplane/src/main.rs

```diff
@@ -1,28 +1,28 @@
-// SPDX-License-Identifier: Apache-2.0
-// Copyright Open Network Fabric Authors
+// // SPDX-License-Identifier: Apache-2.0
+// // Copyright Open Network Fabric Authors
+//
 
-mod args;
-mod config;
-mod nat;
-mod pipeline;
+#![allow(missing_docs)]
+use crate::pipeline::NetworkFunction;
+pub mod args;
+pub mod config;
+pub mod nat;
+pub mod packet;
 
 use crate::args::{CmdArgs, Parser};
-use crate::config::Config;
-use crate::pipeline::{MetaPacket, Metadata, Passthrough, Pipeline};
+pub mod pipeline;
```

Comment:
Why are these public?

Reply:
they (briefly) needed to be public in order to make a benchmark for the rework with the `Chain` struct.  Since we are dropping that whole commit I'm going to fix

Reply:
That said, in a binary crate it make no real difference